### DISCUSSION TITLE
Meta plugin escaping title

### DISCUSSION THREAD

**Discussion by h3** [quasarframework/quasar#9120] [General - Components / Directives / etc] [answered]
I'm using the Meta plugin to set the title of the page dynamically (reactive prop), while it works perfectly fine I found out that when the content of the title contains an ampersand `&` it gets escaped and turn into a `&amp;`.

Most of the time I think that's the desired behavior, however in that case the browser is displaying the `&amp;` in the title bar instead of a `&`.

The only workaround I found to display the title correctly is less than optimal:

```
setTimeout(() => {
  const title = document.querySelector('title')
  title.textContent = this.meta.title.replace('&amp;', '&')
}, 150)
```

I'm wondering if I missed something or if I should file an issue.

**Comment by metalsadman** [comment, marked as answer]
Try writing the string using the unicode of `&` character.

**Reply by h3** [reply]
It does not work with the unicode ampersand, but I found out there's also a "full width" ampersand with which it does works.

I guess it'll do, thanks a bunch!
